docs/getting-started.md

```diff
@@ -0,0 +1,171 @@
+# Getting Started
+
+- [Install](#install)
+- [Create an API](#create-an-api)
+
+Tyk Operator extends Kubernetes API with Custom Resources. API Definitions, Security Policies, Authentication, 
+Authorization, Rate Limits, and other Tyk features can be managed just like other native Kubernetes objects, leveraging 
+all the features of Kubernetes like kubectl, security, API services, RBAC. You can use Tyk Custom Resources to manage 
+and secure REST, TCP, gRPC, GraphQL, and SOAP services. You can even take an existing REST based API and define a GraphQL 
+schema to tell the GraphQL execution engine how to map those REST responses into the GraphQL schema, without a line of code.
+
+This tutorial walks through creating an API using the Tyk Operator.
+You can find all available example ApiDefinition files in the [samples](../config/samples) directory.
+
+## Install
+
+Please follow the [installation documentation](./installation/installation.md) to set up Tyk Operator.
+
+In order to complete this tutorial, you need to have a fully functioning & bootstrapped Tyk installation (CE or Pro Licensed)
+that is accessible from the Kubernetes cluster that will host the Tyk Operator, as explained in the [installation documentation](./installation/installation.md).
+
+## Create an API
+
+Creating an API takes the same approach whether you are using Tyk CE or Tyk Pro. In the first place, an `ApiDefinition` 
+file must be created in the YAML format. Then, Tyk Operator handles the creation of the API.
+
+1. Define your API in the YAML format,
```

Comment:
Should we change to `Define APIDefinition resource in the YAML format`

Reply:
updated 👍